tests/utils/screenshots.js

```diff
@@ -0,0 +1,67 @@
+const sharp = require('sharp');
+
+const MOBILE_WIDTH = 320;
+const MOBILE_HEIGHT = 570;
+const DESKTOP_WIDTH = 1000;
+const DESKTOP_HEIGHT = 820;
+
+const isMobile = async () => {
+  const { width } = await browser.getWindowSize();
+  return width < 768;
+};
+
+const resizeWindowForScreenshots = async () => {
+  if (await isMobile()) {
+    await browser.emulateDevice({
+      viewport: {
+        width: MOBILE_WIDTH,
+        height: MOBILE_HEIGHT,
+        isMobile: true,
+        hasTouch: true,
+      },
+      userAgent: 'my custom user agent'
+    });
+  } else {
+    await browser.setWindowSize(DESKTOP_WIDTH, DESKTOP_HEIGHT);
+  }
+};
+
+const generateScreenshot = async (scenario, step) => {
+  const isCI = process.env.CI === 'true';
+  if (!isCI) {
```

Comment:
Okay with excluding it in [the workflow](https://github.com/medic/cht-core/blob/master/.github/workflows/build.yml). IMO, as soon as we have 1 test case comparing images, it can be included in the workflow